### PR TITLE
Feature/library test

### PR DESCRIPTION
**Pull request by slawler**
Refactor the inner directory from ras-stac to ras_stac to facilitate use as a library as opposed to stand alone utilities.

### CODE REVIEW

**Review by thwllms — COMMENTED** [review]
I'm finding that the `populate` script needs some updates in order to run properly again.

File: ras_stac/utils/data_model.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+class RasStacInputs:
```
[the remark this answers is not in the document]
Comment (sray014) [reply ras_stac/utils/data_model.py:1]: Correct, will soon be used to populate input parameters.

File: .gitignore [hunk 2]
```diff
@@ -136,4 +136,6 @@ scratch/
 
 .data/
 example-data/
-stac-browser/
\ No newline at end of file
+stac-browser/
+
+rashdf/
```
Comment (thwllms) [inline .gitignore:141]: @sray014 is `rashdf` actively used? Or just planned?

File: example.env [hunk 3]
```diff
@@ -2,7 +2,7 @@
 AWS_ACCESS_KEY_ID=user
 AWS_SECRET_ACCESS_KEY=password
 
-# For AWS use your credentials
+# # For AWS use your credentials
```
Comment (thwllms) [inline example.env:5]: Might as well revert this change?

File: populate-sample-data.sh
Comment (thwllms) [inline populate-sample-data.sh:1]: Worked for me this time!

File: ras_stac/utils/s3_utils.py [hunk 4]
```diff
@@ -156,9 +160,43 @@ def init_s3_resources(dev_mode: bool = False):
...
+        resp = s3_client.list_objects_v2(**kwargs)
+        keys += [obj["Key"] for obj in resp["Contents"] if obj["Key"].endswith(suffix)]
+        try:
+            kwargs["ContinuationToken"] = resp["NextContinuationToken"]
+        except KeyError:
+            break
+    return keys
+
+
+import re
```
Comment (thwllms) [inline ras_stac/utils/s3_utils.py:184]: Move `import re` to top of module

File: ras_stac/utils/s3_utils.py [hunk 5]
```diff
@@ -156,9 +160,43 @@ def init_s3_resources(dev_mode: bool = False):
         # Instantitate S3 resources
         session = boto3.Session(
             aws_access_key_id=os.environ.get("AWS_ACCESS_KEY_ID"),
-            aws_secret_access_key=os.environ.get("AWS_ACCESS_KEY_ID"),
+            aws_secret_access_key=os.environ.get("AWS_SECRET_ACCESS_KEY"),
```
Comment (thwllms) [inline ras_stac/utils/s3_utils.py:163]: Hah, good catch!

**Review by thwllms — APPROVED** [review]
@sray014 added a few minor comments, but otherwise LGTM 👍
